src/test/java/com/softserveinc/ita/BasketModalTest.java

```diff
@@ -2,12 +2,15 @@
 
 import com.softserveinc.ita.pageobjects.HomePage;
 import com.softserveinc.ita.pageobjects.ProductPage;
+import com.softserveinc.ita.utils.runners.TestListener;
 import com.softserveinc.ita.utils.runners.TestRunner;
 import org.testng.annotations.DataProvider;
+import org.testng.annotations.Listeners;
 import org.testng.annotations.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@Listeners({TestListener.class})
```

Comment:
instead of attaching a listener to every test, try to attach it only in TestRunner, and share info about reporting in the readme (how to use, generate, and where the report would be placed, and it needs to install something on PC)